tox.ini

```diff
@@ -0,0 +1,58 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+[tox]
+no_package = True
+env_list = lint, unit
+
```

Comment:
Nit: just for consistence with other projects, I would add something like this and use `{all_path}` instead of `.`
```ini
[vars]
src_path = {tox_root}/src
tests_path = {tox_root}/tests
all_path = {[vars]src_path} {[vars]tests_path}
```